packs/feats/floral-restoration.json

```diff
@@ -11,7 +11,7 @@
         },
         "category": "class",
         "description": {
-            "value": "<p><strong>Frequency</strong> once per day</p>\n<hr />\n<p>You request that nearby plants share their vitality with you to replenish your body and magic. You regain 1 Focus Point and [[/r 4d8[healing]]] Hit Points.</p>\n<p>You must be in a location of at least 15 square feet with healthy plant life, though this can be grass, lichen, seaweed, or any other form of naturally occurring flora. Using Floral Restoration on a given section of nature does not harm it, but it prevents that section of nature from giving its vitality to another use of Floral Restoration for 24 hours.</p>\n<p>At 9th level, and every 2 levels thereafter, increase the healing by 1d8.</p>\n<p>@Damage[ceil(@item.level/2)d8]{Leveled Healing}</p>"
+            "value": "<p><strong>Frequency</strong> once per day</p><hr /><p>You request that nearby plants share their vitality with you to replenish your body and magic. You regain 1 Focus Point and @Damage[4d8[healing]] Hit Points.</p>\n<p>You must be in a location of at least 15 square feet with healthy plant life, though this can be grass, lichen, seaweed, or any other form of naturally occurring flora. Using Floral Restoration on a given section of nature does not harm it, but it prevents that section of nature from giving its vitality to another use of Floral Restoration for 24 hours.</p>\n<p>At 9th level, and every 2 levels thereafter, increase the healing by 1d8.</p>\n<p>@Damage[ceil(@item.level/2)d8]{Leveled Healing}</p>"
```

Comment:
```suggestion
            "value": "<p><strong>Frequency</strong> once per day</p><hr /><p>You request that nearby plants share their vitality with you to replenish your body and magic. You regain 1 Focus Point and @Damage[ceil(@item.level/2)d8[healing]] Hit Points.</p>\n<p>You must be in a location of at least 15 square feet with healthy plant life, though this can be grass, lichen, seaweed, or any other form of naturally occurring flora. Using Floral Restoration on a given section of nature does not harm it, but it prevents that section of nature from giving its vitality to another use of Floral Restoration for 24 hours.</p>\n<p>At 9th level, and every 2 levels thereafter, increase the healing by 1d8.</p>"
```